packages/grz-pydantic-models/tests/test_metadata.py

```diff
@@ -66,3 +68,21 @@ def test_file_extensions():
             fileSizeInBytes=0,
             readLength=100,
         )
+
+
+@pytest.mark.parametrize(
+    "case,valid",
+    (
+        ("minimal_consented", True),
+        ("minimal_nonconsented", True),
+        ("mii_ig_consent_v2025_example1", True),
+        ("invalid_missing_fields", False),
+    ),
+)
+def test_research_consent(case: str, valid: bool):
```

Comment:
Should we also add a similar test that checks the actual consent determined for the first three cases?